device/barefoot/x86_64-accton_as9516bf_32d-r0/newport/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,83 @@
+{% set default_cable = '5m' %}
+{% set ingress_lossless_pool_size =  '4194304' %}
```

Comment:
We might have to change these values, Siva should be able to tell us the range of these values for Tofino-2. I guess we might have a little extra buffers to put bump up these values.